generic/tests/netperf.py

```diff
@@ -138,6 +141,22 @@ def mtu_set(mtu):
         for iface in target_ifaces:
             utils.run(host_mtu_cmd % (iface, mtu), ignore_status=False)
 
+    def tweak_tuned_profile():
+        """
+
+        Tweak configuration with truned profile
+
+        """
+        host_tuned_profile = params.get("host_tuned_profile")
+        guest_tuned_profile = params.get("guest_tuned_profile")
+        error_context.context("Changing the MTU of guest", logging.info)
+        if params.get("os_type") == "linux":
+            ssh_cmd(server_ctl, guest_tuned_profile)
+
+        error_context.context("Changing the MTU of client/host", logging.info)
+        ssh_cmd(client, host_tuned_profile)
```

Comment:
I am a little confused here, from L#239 where `client` defined, it actually can be a guest(vm), but here you define the tuned profile `host_tuned_profile`, can we define it `client_tuned_profile` to be clearer? And also for `guest_tuned_profile` has `server_tuned_profile` instead?

Reply:
Update to three parmaeter, server_tuned_profile/client_tuned_profile/host_tuned_profile.